extension/src/background/messageListener/lyraApiMessageListener.ts

```diff
@@ -18,7 +18,7 @@ import { publicKeySelector } from "background/ducks/session";
 
 import { responseQueue, transactionQueue } from "./popupMessageListener";
 
-const WHITELIST_ID = "whitelist";
+const ALLOWLIST_ID = "allowlist";
```

Comment:
I noticed this issue the other day, we redeclare this const in both `lyraApiMessageListener` and `popupMessageListener`. Why don't we make a `background/constants` file and store this there